### ISSUE TITLE
iOS 14 (and betas) Blobs

### ISSUE THREAD

**Issue by anazhd** [airsquared/blobsaver#199]
**Describe the bug**
A clear and concise description of what the bug is and the expected behavior.

**Screenshots**

**Desktop:**
 - OS Version: macOS Big Sur Beta 6
 - iOS Device: iPhone12,1

**Checklist (place an x between the brackets to mark as completed):**
 - [x ] I have made sure I am on the latest version of blobsaver
 - [x ] I have checked the blobsaver wiki to see if there is already a solution to my problem
 - [x ] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
tsschecker = /Applications/blobsaver.app/Contents/MacOS/tsschecker
Extracting build manifest from https://updates.cdn-apple.com/2020SummerFCS/fullrestores/001-46828/6A00C15C-8AEB-490E-A468-04E28C68E7C9/iPhone11,8,iPhone12,1_14.0_18A373_Restore.ipsw
Extracted to /var/folders/h4/lvpk4mt57_78t91xss9qpz9r0000gn/T/BuildManifest1266913018388237549.plist
Running: [/Applications/blobsaver.app/Contents/MacOS/tsschecker, --nocache, -d, iPhone12,1, -s, -e, 618890AE0802E, --save-path, /Users/anazhd/Blobs, --apnonce, 25cf538b4f6f1d38d5a16ff65f03d338c33a35c5fd474b056d8a2bc8d17f35d9, --beta, --buildid, 18A373, -m, /var/folders/h4/lvpk4mt57_78t91xss9qpz9r0000gn/T/BuildManifest1266913018388237549.plist]
Version: b9d193aa6e6d24421094873c830692d02d8b32f5 - 304
libfragmentzip version: 0.59-542a470d7be248681dba71d0f04e7dc8c2718b73
[TSSC] manually specified ECID to use, parsed "618890AE0802E" to dec:1715826732335150 hex:618890ae0802e
[TSSC] manually specified apnonce to use, parsed "25cf538b4f6f1d38d5a16ff65f03d338c33a35c5fd474b056d8a2bc8d17f35d9" to hex:25cf538b4f6f1d38d5a16ff65f03d338c33a35c5fd474b056d8a2bc8d17f35d9
[TSSC] opening /var/folders/h4/lvpk4mt57_78t91xss9qpz9r0000gn/T/BuildManifest1266913018388237549.plist
[Error] [TSSR] Error: could not get RestoreBehavior
[WARNING] [TSSC] could not get id0 for installType=Erase. Using fallback installType=Update since user did not specify installType manually
[Error] [TSSR] Error: could not get RestoreBehavior
[Error] [TSSR] Error: could not get id0 for installType=Update
[Error] [TSSR] failed to build tss request

[Error] [TSSC] checking tss status failed!

```

Couldn't generate blobs for iOS 14 (including betas). I'm not sure if it's already supported, I'm putting it here because i was ootl. Not a major concern, just incase I missed any announcement or nobody reported it yet.

**Comment by Yoshi24517** [comment]
Yeah I sent a reddit PM about this exact thing to him earlier, he knows and is working on a fix! :D

**Comment by airsquared** [comment]
Please try this build:

- [blobsaver-macos.dmg](https://github.com/airsquared/blobsaver/files/5248204/blobsaver-macos.dmg.zip)
- [blobsaver-windows.exe](https://github.com/airsquared/blobsaver/files/5248214/blobsaver-windows.exe.zip)
- [blobsaver-linux.tar.gz](https://github.com/airsquared/blobsaver/files/5248218/blobsaver-linux.tar.gz)

**Comment by piguin** [comment]
thanks. this build works for my iPhone XR, which consistently fails to save blobs with 14.0 firmware with v2.5.3

**Comment by KalooNie** [comment]
> Please try this build:
>     * [blobsaver-windows.exe](https://github.com/airsquared/blobsaver/files/5248214/blobsaver-windows.exe.zip)

Worked on my iPhone XR as well. Saved 13.7 & 14.0 successfully! Thank you!
(Was failing on 2.5.3. Kept saying "check board configuration".)

**Comment by airsquared** [comment]
This has been fixed in v2.5.4
